setup.py

```diff
@@ -1,35 +1,62 @@
 #!/usr/bin/env python
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 
-# NOTE: The configuration for the package, including the name, version, and
```

Comment:
Can we keep an edited version of this note?

Reply:
Done